docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.1'
+
+networks:
+  db-data-net:
+    driver: bridge
+
+
+services:
+
+  db:
+    image: fredisson11/mysql-local:1.0.0
+    container_name: db-container
+    restart: always
+    ports:
+      - "3306:3306"
+    volumes:
+      - db_data:/var/lib/mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: example
+      MYSQL_DATABASE: app_db
+      MYSQL_USER: app_user
+      MYSQL_PASSWORD: 1234
+    networks:
+      - db-data-net
+
+  todo-app:
+    build: ./
```

Comment:
The `build` directive for the `todo-app` service is redundant since an image is already specified. Decide whether you want to use a pre-built image or build from a Dockerfile, and adjust accordingly.